eli5/formatters/html.py

```diff
@@ -27,13 +28,15 @@
     remaining_weight_color=lambda ws, w_range, pos_neg:
         format_hsl(remaining_weight_color_hsl(ws, w_range, pos_neg)),
     format_feature=lambda f, w, hl: _format_feature(f, w, hl_spaces=hl),
+    format_value=format_value,
     format_decision_tree=lambda tree: _format_decision_tree(tree),
 ))
 
 
 def format_as_html(explanation, include_styles=True, force_weights=True,
                    show=fields.ALL, preserve_density=None,
-                   highlight_spaces=None, horizontal_layout=True):
+                   highlight_spaces=None, horizontal_layout=True,
+                   show_feature_values=True):
```

Comment:
If this is True by default we should also update all IPython notebooks.

By the way, what do you think about keeping it False? Values can be good to see in many cases, but they are also confusing and require more explanation and more classifier knowledge (what's the difference between Value and Contribution?). For text explanations what users get with this option set to true is a table like this above the highlighted text:

![table](https://cloud.githubusercontent.com/assets/107893/21572809/c6550c28-cf07-11e6-8e1d-372d8fa76f48.png)

It could be also good to add title attribute to table header, to show some help on what these columns mean. For example, it is not clear if user should multiply Contribution by Value to get the importance or not.